packages/babel-plugin-proposal-object-rest-spread/test/fixtures/object-rest/variable-destructuring/output.js

```diff
@@ -10,10 +10,11 @@ var {
 } = z,
     y1 = babelHelpers.objectWithoutProperties(z, ["x1"]);
 x1++;
-var {
-  [a]: b
+var _a = a,
```

Comment:
is it necessary to split this computed property like this?

Reply:
It's a bug. I'm looking at it right now